config/sidekiq-schedule.yml

```diff
@@ -7,3 +7,8 @@ fetch_and_sync_all_pull_requests_reputation:
   cron: "0 1 * * 1"
   class: "FetchAndSyncAllPullRequestsReputationJob"
   queue: dribble
+
+sync_issues:
+  cron: "0 2 * * 1"
```

Comment:
I've started this job one hour after the other job, which is enough time for it to finish I think.

Reply:
is it worth doing on a different day so that we don't overload our token?

Reply:
Good idea

Reply:
It now runs a day later. Theoretically it shouldn't really matter as tokens reset each hour, but it is better to spread these jobs around regardless.